.github/workflows/android-build-test.yml

```diff
@@ -0,0 +1,19 @@
+name: Test android build
+on:
+  pull_request:
+    branches: [master]
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: Use Node.js 14
+        uses: actions/setup-node@v2
+        with:
+          node-version: 14
+          cache: 'yarn'
+      - name: Install dependencies
+        run: yarn && cd TestsExample && yarn
```

Comment:
Why is `yarn` run only here in the main dir of `react-native-screens` and omitted in other builds? Are we certain that this job will run first and packages will be cached for others? It so I'd still split it to separate steps.

Reply:
It was there because before I was also running emulator and the app crashed without root dependencies. 
I decided that it is not necessary to run it (without emulator job takes 3 minutes instead of 5 and can be ran on ubuntu).

Reply:
I removed the unnecessary `yarn`